build-system/tasks/extension-helpers.js

```diff
@@ -656,20 +670,7 @@ async function buildExtensionJs(extDir, name, version, latestVersion, options) {
       toName: `${name}-${version}.max.js`,
       minifiedName: `${name}-${version}.js`,
       latestName: latest ? `${name}-latest.js` : '',
-      // Wrapper that either registers the extension or schedules it for
-      // execution after the main binary comes back.
-      // The `function` is wrapped in `()` to avoid lazy parsing it,
-      // since it will be immediately executed anyway.
-      // See https://github.com/ampproject/amphtml/issues/3977
```

Comment:
This refers to the included function in the extension payload:
```js
f: (function(AMP, _){...})
```
The parentheses were lost at some point of changing the building pipeline. On any currently published extension binary, I'll see:
```js
f:function(AMP,_){...}
```
Is the function lazy building when it lacks parens a real issue? If so, we should reopen `#3977`